graylog2-web-interface/webpack.config.js

```diff
@@ -44,19 +44,6 @@ const BABELLOADER = { loader: 'babel-loader', options: BABELOPTIONS };
 // eslint-disable-next-line import/no-dynamic-require
 const BOOTSTRAPVARS = require(path.resolve(ROOT_PATH, 'public', 'stylesheets', 'bootstrap-config.json')).vars;
 
-const getCssLoaderOptions = () => {
```

Comment:
I think we could keep this function. The culprit seems to be `css-loader` defaulting to `mode: 'local'`, that is why the `modules: 'global'` (which is an abbreviation for `modules: { mode: 'global' }` fixes it. Maybe we can keep `localIdentName` for dev and just return `mode: 'global'` for all modes?

Reply:
Picked it up in the last commit. I agree with the solution 😊